src/main/java/dev/slimevr/gui/SkeletonConfigGUI.java

```diff
@@ -93,8 +93,9 @@ float proportionsIncrement(Boolean negative) {
 		return increment;
 	}
 
-	String getBoneLengthString(SkeletonConfigValue joint) { // Rounded to the
-															// nearest 0.5
+	String getBoneLengthString(SkeletonConfigOffsets joint) { // Rounded to
+																// the
+		// nearest 0.5
```

Comment:
Lol what's up with this comment?

Reply:
automatic formatting go brrr.
Old GUI code will get deleted anyway and as you said (on Discord for anyone reading) it doesn’t matter so I’m gonna resolve this